### DISCUSSION TITLE
[SUGESTÃO[FEEDBACK] Uma branch secundaria para fazer fork.

### DISCUSSION THREAD

**Discussion by xafixav** [digitalinnovationone/dio-lab-open-source#42511] [Ideias e Sugestões]
Seria interessante existir uma branch secundaria "limpa" para fazer o fork, creio que em poucos meses ou anos vai ficar bem gigante o tamanho da main desse projeto.

## Seguem algumas prints de hoje 29/07/2024:

![Screenshot 2024-07-29 172104](https://github.com/user-attachments/assets/067f40ff-dee2-4f19-b65e-b6e67b307d13)
![image](https://github.com/user-attachments/assets/3d19992e-60da-40bb-a623-137d7f2d5cbc)

**Comment by Luciano-gOliveira** [comment]
up

**Comment by kBruna** [comment]
Dia 16/09:
Arquivos: 25821
Tamanho: 355.25MiB

**Reply by odevlucas** [reply]
Dia: 05/11
Arquivos: 28.153
Tamanho: **1.1GB**

O que houve por aqui?

**Reply by xafixav** [reply]
@elidianaandrade Parece que ta ficando gigante...

**Comment by elidianaandrade** [comment]
Olá, @xafixav! Obrigada por sinalizar e pela sugestão! Recebo muuuitas notificações aqui no GitHub e algumas acabam passando. Criei aqui e nomeei como "template".
